llvm/lib/CodeGen/BranchRelaxation.cpp

```diff
@@ -384,11 +391,17 @@ bool BranchRelaxation::fixupConditionalBranch(MachineInstr &MI) {
 
   auto finalizeBlockChanges = [&](MachineBasicBlock *MBB,
                                   MachineBasicBlock *NewBB) {
-    // Keep the block offsets up to date.
-    adjustBlockOffsets(*MBB);
+    if (NewBB == nullptr)
+      return;
```

Comment:
nit: now `MBB` is not used and `finalizeBlockChanges` processes block changes around `NewBB` (including its prev and next). Remove unused `MBB` parameter and optionally rename `finalizeBlockChanges` to something like `finalizeBlockChangesForNewBB` to make code around the call-site clearer.